docs/UserGuide.adoc

```diff
@@ -210,12 +213,96 @@ Updates the winner of a particular prize category
 Lists the prize categories and their winners (if any)
 
 
-=== Bulk Registration: `addBulk PATH_TO_CSV_FILE`
-You may add multiple participants at once into Alfred through the specification of a .csv file
+=== Bulk Registration: `import fp/PATH_TO_CSV_FILE`
+You may import multiple entities at once into Alfred through the specification of a CSV file.
+
+Example:
+
+* `import fp/C:/User/Hackathon2019/participant.csv` will import data from the participant.csv file into Alfred.
+* `import fp/Hackathon2019/participant.csv` will look for the CSV file in your current directory (or the folder where alfred.jar is downloaded).
+
+==== CSV File Formatting
+In order for the contents of the CSV file to be correctly loaded into Alfred, the file has to be in the correct format.
+
+****
+`EntityType` - single capital letter representing the entity type
+
+* **Mentor:** M
+* **Participant:** P
+* **Team:** T
+
+`ID` - the ID of the entity (optional)
+
+* **Mentor:** M-`d`, where `d` can be any positive integer
+* **Participant:** P-`d`, where `d` can be any positive integer
+* **Team:** T-`d`, where `d` can be any positive integer
+** You can also express `ID` as only a positive integer.
+
+`Name`, `Organization`, `ProjectName` - can be any combination of spaces, letters, and these special characters (,.-')
+
+`Phone` - can be any combination of numbers and country code may be included
+
+`Email` - must include an address and an email domain
+
+`SubjectName` - the subject a mentor or team will be focusing on in the Hackathon +
+Must be one of the values below:
+
+* Environmental
+* Social
+* Health
+* Education
+
+`Score` - must be an integer ranging from 0 to 100
+
+`Location` - must be an integer ranging from 0 to 1000
+****
+
+===== Mentor Data
+Header: `EntityType,ID,Name,Phone,Email,Organization,SubjectName`
+
+Example:
+
+* M,M-1,Alfred the Mentor,12345678,alfred@batcave.com,Batcave Corp.,Education
+* M,1,Alfred the Mentor,+6512345678,alfred@batcave.com,Batcave Corp.,EDUCATION
+* M,,Alfred the Mentor,+65 12345678,alfred@batcave.com,Batcave Corp.,Education
```

Comment:
Hahaha just nit picking here, you may want to include space after each comma as it seems quite hard to read :)

Reply:
Oh i think the standard for csv file separator is just one comma without spaces